.circleci/config.yml

```diff
@@ -294,3 +317,24 @@ jobs:
                 "body": "Link to failed job: '"${CIRCLE_BUILD_URL}"'."
                 }'
           when: on_fail
+
+  integration-tests:
```

Comment:
Are using the machine executor because we want to be able to run docker? I thought enabling remote docker execution context on docker executor allowed to spawn docker containers from regular executors as well.

Reply:
Yeah you can spawn docker containers but you can't connect to them directly. See https://circleci.com/docs/2.0/building-docker-images/#accessing-services

> It is not possible to start a service in remote docker and ping it directly from a primary container or to start a primary container that can ping a service in remote docker.